Mathlib/Logic/Basic.lean

```diff
@@ -247,8 +247,11 @@ classical ones, as these may cause instance mismatch errors later.
 /-- The Double Negation Theorem: `¬ ¬ P` is equivalent to `P`.
 The left-to-right direction, double negation elimination (DNE),
 is classically true but not constructively. -/
-@[simp] theorem not_not : ¬¬a ↔ a := Decidable.not_not
-#align not_not not_not
+add_decl_doc Classical.not_not -- TODO: move to Std
```

Comment:
You cannot add a docstring for an imported declaration, this doesn't do anything but trigger a panic. I would suggest building mathlib with `LEAN_ABORT_ON_PANIC=1` on CI.